.hintrc

```diff
@@ -14,5 +14,6 @@
       "meta-charset-utf-8",
       "meta-viewport",
       "no-inline-styles:error"
-    ]
+    ],
```

Comment:
[OPTIONAL] I would highly suggest you make your project as clean as possible. Kindly, remove the .vscode file from your project.

<img width="933" alt="Screen Shot 2022-04-16 at 5 23 21 PM" src="https://user-images.githubusercontent.com/69011963/163675051-3c2114b3-9f65-4d1d-aa0d-dbb6df3cb5a6.png">